developer-docs-site/docs/tools/aptos-cli/use-cli/use-aptos-ledger.md

```diff
@@ -0,0 +1,94 @@
+[3] Derivation path: m/44'/637'/3'/0'/0' (Address: 2dcf037a9f31d93e202c074229a1b69ea8ee4d2f2d63323476001c65b0ec4f31)
+[4] Derivation path: m/44'/637'/4'/0'/0' (Address: 23c579a9bdde1a59f1c9d36d8d379aeefe7a5997b5b58bd5a5b0c12a4f170431)
+0
+Account 59836ba1dd0c845713bdab34346688d6f1dba290dbf677929f2fc20593ba0cfb has been already found onchain
+
+---
+Aptos CLI is now set up for account 59836ba1dd0c845713bdab34346688d6f1dba290dbf677929f2fc20593ba0cfb as profile myprofile!  Run `aptos --help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+In the above, we have created a new profile called `myprofile` and have chosen to use the first Ledger account (index 0) to sign transactions. If there is a certain index account you would like to use, you are welcome to use it.
+
+
+After the above command, a new profile will be created in `~/.aptos/config.yml` and will look like the following:
+```yaml
+  myprofile:
+    public_key: "0x05a8ace09d1136181029be3e817de3619562b0da2eedbff210e2b2f92c71be70"
+    account: 59836ba1dd0c845713bdab34346688d6f1dba290dbf677929f2fc20593ba0cfb
+    rest_url: "https://fullnode.devnet.aptoslabs.com"
+    faucet_url: "https://faucet.devnet.aptoslabs.com"
+    derivation_path: "m/44'/637'/0'/0'/0'"
+```
+Notice that the above stores the derivation path instead of private key. This is because the private key is stored on your Ledger device, and is never exposed to the `aptos` tool.
+
+## Publish a package with Ledger
+Once you have created a profile, you can use it to publish a package. The `aptos` tool will prompt you to confirm the transaction on your Ledger device.
+Note: Make sure that you are on the same directory as where your move module is located:
+```bash
+$ aptos move publish --profile myprofile --named-addresses hello_blockchain=myprofile
+Compiling, may take a little while to download git dependencies...
+INCLUDING DEPENDENCY AptosFramework
+INCLUDING DEPENDENCY AptosStdlib
+INCLUDING DEPENDENCY MoveStdlib
+BUILDING Examples
+package size 1755 bytes
+Do you want to submit a transaction for a range of [139600 - 209400] Octas at a gas unit price of 100 Octas? [yes/no] >
+yes
+{
+  "Result": {
+    "transaction_hash": "0xd5a12594f85284cfd5518d547d084030b178ee926fa3d8cbf699cc0596eff538",
+    "gas_used": 1396,
+    "gas_unit_price": 100,
+    "sender": "59836ba1dd0c845713bdab34346688d6f1dba290dbf677929f2fc20593ba0cfb",
+    "sequence_number": 0,
+    "success": true,
+    "timestamp_us": 1689887104333038,
+    "version": 126445,
+    "vm_status": "Executed successfully"
+  }
+}
+```
+
+After the above command, you will be prompted to confirm the transaction on your Ledger device. Once you confirm, the transaction will be submitted to the network. Note: Make sure you have `Blind Signing` enabled on your Ledger device. Otherwise you will not be able to sign transactions.
+
+## Common Errors:
+
+#### Error: Wrong raw transaction length
```

Comment:
```suggestion
### Error: Wrong raw transaction length
```

Reply:
For this one, I actually like the `####` size better 😄

Reply:
Let's leave the indenting to be one level below (even if the size is better)